device-registry/src/test/scala/org/genivi/sota/device_registry/db/DeviceRepositorySpec.scala

```diff
@@ -0,0 +1,35 @@
+/**
+  * Copyright: Copyright (C) 2016, ATS Advanced Telematic Systems GmbH
+  * License: MPL-2.0
+  */
+
+package org.genivi.sota.device_registry.db
+
+import org.genivi.sota.core.DatabaseSpec
+import org.genivi.sota.data.DeviceGenerators.{genDeviceId, genDeviceT}
+import org.genivi.sota.data.Namespaces
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{FunSuite, ShouldMatchers}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class DeviceRepositorySpec extends FunSuite
+                           with DatabaseSpec
+                           with ScalaFutures
+                           with ShouldMatchers {
+
+  test("activatedAt") {
```

Comment:
Think this should be more specific, ie, "activatedAt is updated after `setActivated`"